assets/js/frontend/single-product.js

```diff
@@ -228,10 +261,26 @@ jQuery( function( $ ) {
 			};
 
 			// Initializes and opens PhotoSwipe.
-			var gallery = new PhotoSwipe( pswpElement, PhotoSwipeUI_Default, items.items, options );
+			var gallery = new PhotoSwipe( pswpElement, PhotoSwipeUI_Default, items, options );
 			gallery.init();
-		}
+		};
+
+		this.init();
+	};
+
+	/**
+	 * Function to call wc_product_gallery on jquery selector.
+	 */
+	$.fn.wc_product_gallery = function( args ) {
+		new Product_Gallery( this, args );
+		return this;
 	};
 
-	wc_product_gallery.init();
-});
+	/*
+	 * Initialize all galleries on page.
+	 */
+	$( '.woocommerce-product-gallery' ).each( function() {
+		$( this ).wc_product_gallery();
+	} );
```

Comment:
Initialize product gallery functionality on whatever `.woocommerce-product-gallery` elements exist on the page. Third parties may do the same for other elements, e.g. dynamically added ones.